src/lib/chips/chip-input.ts

```diff
@@ -38,6 +38,7 @@ let nextUniqueId = 0;
     '(focus)': '_focus()',
     '(input)': '_onInput()',
     '[id]': 'id',
+    '[attr.placeholder]': 'placeholder || null',
```

Comment:
@tinayuangao should we deprecate the `placeholder` input for 7.0? It doesn't do anything, apart from proxying the value to the `placeholder` attribute. Consumers can achieve the same by binding to `attr.placeholder`.

Reply:
Yes, I think we can